README.md

```diff
@@ -472,6 +472,15 @@ opens your files. The default is `utf-8`.
 IMAGE_PROCESS_ENCODING = "utf-8"
 ```
 
+#### Copying EXIF tags
+
+You may ask `image_process` to copy the EXIF tags from your original image to
+the transformed images. You must have [exiftool](https://exiftool.org/) installed.
```

Comment:
For consistency, I suggest changing: \`image_process\` ⇢ `_Image Process_`

Reply:
fixed